db/data/queries/insertOrder.sql

```diff
@@ -1,9 +1,8 @@
 DESCRIBE `Order`;
```

Comment:
can you insert **USE `quickOrder`;** as the first line?  i think you deleted it by mistake before